tensor2tensor/data_generators/text_encoder.py

```diff
@@ -73,7 +78,7 @@ def decode(self, ids):
     decoded_ids = []
     for id_ in ids:
       if 0 <= id_ < self._num_reserved_ids:
-        decoded_ids.append(RESERVED_TOKENS[int(id_)])
```

Comment:
should keep the int cast because the ids coming in may be numpy ints which cannot be used for indexing.